src/screens/sgapeople.js

```diff
@@ -3,7 +3,577 @@ import React, { Component } from 'react';
 class SGAPeople extends Component {
   render() {
     return (
+      <div id="content">
+        <div class="clearfix">
+          <div class="ui-overlay" id="help" style="display:none;">
+            <div class="ui-widget-overlay"></div>
+            <div class="ui-corner-all" id="overlay" style="width: 100%; height: 600px; position: absolute;overflow-y:scroll; margin-top:10px;">
+              <button onclick="closeHelp()" style="float:right;" type="submit">X</button>
+              <div class="helppage" style="margin-left: 10px; margin-right:10px;">
+                <h1>Submitting SGA Bills</h1>
+                <h2>Introduction</h2>
+                <p>The following will help you to prepare and to learn the process that is required to obtain funding from, or pass a resolution with, the Student Government Association (SGA) at Georgia Tech.<br> The Student Government’s legislative structure consists of a body of undergraduate students (Undergraduate House of Representatives or UHR) and of a body of graduate students (Graduate Student Senate or GSS). The passing of a “joint” bill requires consideration of a bill in each body and is subject to passing by an “enactment ratio.” Mostly all financial requests by student organizations will be classified as a joint bill. The enactment ratio is a mathematical calculation to determine whether a bill passes both bodies of SGA. It ensures that undergraduate and graduates are represented proportionally since undergraduates represent more of the student body and contribute more money to SGA from the Student Activity Fee.<br> The money that SGA allocates to organizations comes from the “Student Activity Fee” line that shows up on the tuition and fee assessment paid by each student every semester. This money is then budgeted to many different organizations with the Student Center and the Campus Recreation Center receiving approximately 70% of that budget. At the end of the year, the money that has not been spent is placed into several accounts that can then be allocated to different groups for special one-time items. Student Government has a set of policies and priorities that it follows with regards to funding requests.<br> Please keep in mind that SGA has a set amount of money that it can spend in one year. SGA attempts to wisely spend that set of money on the organizations with most demonstrated need and on those bills that will benefit the majority of campus. Not every request will end in success.</p>
```

Comment:
Also do a replace all to replace `<br>` with `<br/>`. JSX doesn't automatically close self closing elements for some reasone.